pkg/vpcmodel/explainabilityConnectivity.go

```diff
@@ -29,9 +29,30 @@ type rulesConnection struct {
 	egressRules  rulesInLayers
 }
 
-// loadBalancerDennyEgressRule - is a rule applied to all private IPs of a given load balancer:
+// the load balancer is a rule applied to all private IPs of a given load balancer:
 // these private IPs can only connect to pool members of the load balancer
-type loadBalancerDenyEgressRule bool
+type LoadBalancerRule struct {
+	// the relevant load balancer:
+	lb LoadBalancer
+	//	Denny- true if src is pip, and dst is not pool member:
+	denny bool
+}
+
+func NewLoadBalancerRule(lb LoadBalancer, denny bool) *LoadBalancerRule {
+	return &LoadBalancerRule{lb, denny}
+}
+func (lbr *LoadBalancerRule) Denny() bool { return lbr.denny }
+func (lbr *LoadBalancerRule) StringDetailsOfRule() string {
```

Comment:
This should surely not be here. 
If it is not in the IBM package then at least put in in `explainabilityPrint.go`

Reply:
decided to make it an interface, with implementation in the ibmvpc.